cernopendata_client/cli.py

```diff
@@ -257,3 +257,79 @@ def download_files(server, recid, doi, title, protocol, expand, name, regexp, ra
         download_single_file(path=path, file_location=file_location)
 
     click.echo("\n==> Success!")
+
+
+@cernopendata_client.command()
+@click.option("--recid", type=click.INT, help="Record ID")
+@click.option(
+    "--server",
+    default="http://opendata.cern.ch",
+    type=click.STRING,
+    help="Which CERN Open Data server to query? [default=http://opendata.cern.ch]",
+)
+def verify_files(server, recid):
+    """Verify downloaded data file integrity.
+
+    Select a CERN Open Data bibliographic record by a record ID and
+    verify integrity of downloaded data files belonging to this
+    record.
+
+    \b
+    Examples:
+      $ cernopendata-client verify-files --recid 5500
+    """
+
+    from .verifier import get_file_info_local
+    from .search import get_file_info_remote
```

Comment:
Just curious, why this is in-function import?

Reply:
Yes, I can move them "upstairs".